docs/src/content/docs/get-started/realtime-leaderboard.mdx

```diff
@@ -0,0 +1,208 @@
+---
+title: "Building a Realtime Leaderboard with DiceDB"
```

Comment:
Just a suggestion: I feel like `with DiceDB` is redundant. I tried the following
```suggestion
title: "Build a Realtime Leaderboard"
```
<img width="1428" alt="image" src="https://github.com/user-attachments/assets/2c47228a-ece7-44b7-b601-58ead84867f3">